swap_meet/vendor.py

```diff
@@ -1,2 +1,110 @@
+from .item import Item
+
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self,inventory = None):
+        if inventory is None:
+            inventory = []
+        self.inventory = inventory
+
+    def add(self,item_to_added):
+        self.inventory.append(item_to_added)
+        return item_to_added
+    
+    def remove(self,item_to_removed):
+        if item_to_removed not in self.inventory:
+            return False
+
+        self.inventory.remove(item_to_removed)
+        return item_to_removed
+
+    def get_by_id(self, id):
+        #loop through the item in inventory
+        #if id input is equal to the item id in inventory then return item
+        for item in self.inventory:
+            if id == item.id:
+                return item
+        return None
+    
+    def swap_items(self, other_vendor, my_item, their_item):
+        if my_item not in self.inventory or their_item not in other_vendor.inventory:
+            return False
+        
+        self.remove(my_item)
+        other_vendor.add(my_item)
+
+        other_vendor.remove(their_item)
+        self.add(their_item)
```

Comment:
These statements look good! If we wanted to refactor and streamline the code just a little bit, we could do so by remembering that the `remove()` function returns the item that has been removed! We can then nest it inside our add statement to both add and remove on one line! I've included an original part of your code along with a new snippet!

```suggestion
        self.remove(my_item)
        other_vendor.add(my_item)

        self.add(other_vendor.remove(their_item))
```